IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedBldgElements/Entities/IfcDoorType/Documentation.md

```diff
@@ -1,34 +1,48 @@
-It is used to define a door specification, or door style (i.e. the specific product information that is common to all occurrences of that door type). Door types may be exchanged without being already assigned to occurrences.  
-NOTE The product representations are defined as representation maps (at the level of the supertype _IfcTypeProduct_, which gets assigned by an element occurrence instance through the _IfcShapeRepresentation.Item[1]_ being an _IfcMappedItem_.  
-Occurrences of the _IfcDoorType_ within building models are represented by instances of _IfcDoor_ or _IfcDoorStandardCase_.  
-******Operation type use definition**  
-The _IfcDoorTypeOperationEnum_ defines the general layout of the door type and its symbolic presentation. Depending on the enumerator, the appropriate instances of _IfcDoorLiningProperties_ and _IfcDoorPanelProperties_ are attached in the list of _HasPropertySets_. The _IfcDoorTypeOperationEnum_ mainly determines the hinge side (left hung, or right hung), the operation (swinging, sliding, folding, etc.) and the number of panels.  
-NOTE There are different definitions in various countries on what a left opening or left hung or left swing door is (same for right). Therefore the IFC definition may derivate from the local standard and may need to be mapped appropriately.  
-See geometry use definitions at _IfcDoorTypeOperationEnum_ for the correct usage of opening symbols for different operation types.  
-  
-**Material Use Definition**  
-The material of the _IfcDoorType_ is defined by the _IfcMaterialConstituentSet_ or as fall back by _IfcMaterial_ and attached by the _IfcRelAssociatesMaterial_._RelatingMaterial_. It is accessible by the inverse _HasAssociations_ relationship.  
-The following keywords for _IfcMaterialConstituentSet.MaterialConstituents[n].Name_ shall be used:  
+A door type defines the particular parameter of the lining and one (or several) panels through the _IfcDoorLiningProperties_ and the _IfcDoorPanelProperties_ as predefined property sets applicable to doors only.
+
+It is used to define a door specification, or door style (i.e. the specific product information that is common to all occurrences of that door type). Door types may be exchanged without being already assigned to occurrences.
+
+> NOTE&nbsp; The product representations are defined as representation maps (at the level of the supertype _IfcTypeProduct_, which gets assigned by an element occurrence instance through the _IfcShapeRepresentation.Item[1]_ being an _IfcMappedItem_.
+
+Occurrences of the _IfcDoorType_ within building models are represented by instances of _IfcDoor_ or _IfcDoorStandardCase_.
+
+> HISTORY&nbsp; New entity in IFC4. The entity _IfcDoorType_ replaces the previous definition _IfcDoorStyle_ (which is deprecated in IFC4).
+
+{ .use-head}
+Operation type use definition
```

Comment:
What is this? This is 'usage' and has no place here. I propose to remove it from the entity definition.
```suggestion

```